dev/com.ibm.ws.jdbc_fat_oracle/publish/servers/com.ibm.ws.jdbc.fat.oracle.ucp/server.xml

```diff
@@ -0,0 +1,87 @@
+      <feature>jdbc-4.2</feature>
+      <feature>jndi-1.0</feature>
+      <feature>componenttest-1.0</feature>
+    </featureManager>
+    
+    <include location="../fatTestPorts.xml"/>
+
+    <application location="oracleucpfat.war" >
+        <classloader commonLibraryRef="UCPDBLib"/>
+    </application>
+    
+    <library id="UCPDBLib">
+    	<fileset dir="${shared.resource.dir}/ucp"/>
+    </library>
+    
+    <connectionManager id="conMgr" maxPoolSize="2" enableSharingForDirectLookups="false"/> 
+    
+    <connectionManager id="conMgrUpdate" maxPoolSize="1" />
+    
+    <connectionManager id="sharedConMgr" maxPoolSize="1" />
+    
+    <!-- This DataSource intentionally uses a connectionManagerRef -->
+    <dataSource id="ucpDS" jndiName="jdbc/ucpDS" connectionManagerRef="conMgr" validationTimeout="20" >
+    	<jdbcDriver libraryRef="UCPDBLib" />
+    	<properties.oracle.ucp maxStatements="5"  maxPoolSize="3" user="${env.USER}" password="${env.PASSWORD}" connectionWaitTimeout="30" URL="${env.URL}" />
+    </dataSource>
+    
+    <dataSource id="ucpDSSameConMgr" jndiName="jdbc/ucpDSSameConMgr" connectionManagerRef="sharedConMgr" validationTimeout="20" >
+    	<jdbcDriver libraryRef="UCPDBLib" />
+    	<properties.oracle.ucp URL="${env.URL}" user="${env.USER}" password="${env.PASSWORD}" />
+    </dataSource>
+    
+    <dataSource id="ucpXADS" jndiName="jdbc/ucpXADS" type="javax.sql.XADataSource">
+    	<jdbcDriver libraryRef="UCPDBLib" />
+    	<properties.oracle.ucp maxPoolSize="3" connectionWaitTimeout="30" URL="${env.URL}" user="${env.USER}" password="${env.PASSWORD}" />
+    </dataSource>
+    
+    <dataSource id="ucpConnectionPoolDS" jndiName="jdbc/ucpConnectionPoolDS" type="javax.sql.ConnectionPoolDataSource" >
+    	<jdbcDriver libraryRef="UCPDBLib" />
+    	<properties.oracle.ucp URL="${env.URL}" user="${env.USER}" password="${env.PASSWORD}" />
+    </dataSource>
+    
+    <dataSource id="ucpDriverDS" jndiName="jdbc/ucpDriverDS" type="java.sql.Driver" >
+    	<jdbcDriver libraryRef="UCPDBLib" />
+    	<properties.oracle.ucp maxPoolSize="3" connectionWaitTimeout="30" URL="${env.URL}" user="${env.USER}" password="${env.PASSWORD}"/>
+    </dataSource>
+    
+    <dataSource id="ucpDSAuthData" jndiName="jdbc/ucpDSAuthData">
+   	    <jdbcDriver libraryRef="UCPDBLib" />
+   		<properties.oracle.ucp URL="${env.URL}" user="${env.USER}" password="${env.PASSWORD}"/>
+   		<containerAuthData user="wrong" password="wrong" />
+    </dataSource>
+    
+    <!-- This DataSource intentionally uses the generic properties element and embedded connection manager -->
+    <dataSource id="ucpDSEmbeddedConMgr" jndiName="jdbc/ucpDSEmbeddedConMgr" internalDevNonshipFunctionDoNotUseProduction="true" type="javax.sql.DataSource" validationTimeout="20" >
+    	<connectionManager maxPoolSize="2" minPoolSize="1"/> 
+    	<jdbcDriver libraryRef="UCPDBLib" javax.sql.DataSource="oracle.ucp.jdbc.PoolDataSourceImpl"/>
+    	<!-- TODO remove noship guard -->
```

Comment:
Is this comment still valid? I think the noship guard attr it used to refer to has since been removed